jobs/build/promote/Jenkinsfile

```diff
@@ -333,6 +333,20 @@ node {
                     errata_url = retval.errataUrl
                 }
             }
+            stage("add cve flaw bugs") {
+                if (params.DRY_RUN || advisory == -1) {
+                    return
+                }
+                if (arch != 'x86_64') {
+                    return
+                }
+                if (major == 4 && !is_4stable_release) {
+                    return
+                }
+                ["rpm", "image", "extras"].each {
+                    script: "${buildlib.ELLIOTT_BIN} -g ${group} attach-cve-flaws --use-default-advisory ${it}"
+                }
```

Comment:
Sometimes we don't have rpm or extras advisories for a release. I'll prefer to only attach flaw bugs to the main image advisory, or ignore the errors if those advisories don't exist or got closed.

Reply:
this is a good point because actually the only problem here is with the payload advisory; its URL gets written into the payload image, so we need it to be correct at that time. the other ones can change without any problem.

i think the goal is to automate all this, so i like running it on all (BTW we could see bugs attached to the metadata advisory at some point also). but to yuxiang's point some advisories might not be shipping, so i'd probably wrap each in a retry/skip to query the human if it doesn't work.

Reply:
... which is almost what you updated it with, great :) i think i would do the `retrySkipAbort` on each individual advisory though (or perhaps run in parallel and wrap that?)